articles/iot-edge/how-to-deploy-cli-at-scale.md

```diff
@@ -162,6 +162,23 @@ The previous example showed a layered deployment setting the `properties.desired
 }
 ```
 
+The same can also be expressed with:
+
+```json
+"SimulatedTEmperatureSensor": {
+  "properties.desired.layeredProperties.SendData" : true,
+  "properties.desired.layeredProperties.SendInterval": 5
+}
+```
+
+> Warning: Currently there is a bug with a layered deployment that updates only properties, where the deployment is never marked as applied in the portal
```

Comment:
> **Warning**: Currently there is a bug with a layered deployment that updates only properties.
These layers need an empty edgeAgent object (i.e.`"$edgeAgent":{}`) in the deployment manifest to be considered a valid layered deployment. 
They will also **only** shown as applied in the module twins they target. For the `$edgeAgent` module twin, and hence in the Azure portal, the layer will be marked as targeted.